TG.Backend/Middlewares/ErrorHandlingMiddleware.cs

```diff
@@ -0,0 +1,24 @@
+using FluentValidation;
+
+namespace TG.Backend.Middlewares;
+
+public class ErrorHandlingMiddleware : IMiddleware
+{
+    public async Task InvokeAsync(HttpContext context, RequestDelegate next)
+    {
+        try
+        {
+            await next.Invoke(context);
+        }
+        catch (ValidationException validationException)
+        {
+            context.Response.StatusCode = 400;
```

Comment:
Same here, there is also another option (that contains values that are int already - no cast needed):
<img width="509" alt="image" src="https://user-images.githubusercontent.com/48764491/231884109-954fa0a0-b051-4d4b-958a-58d943999951.png">